tests/test_app_vip_comprehensive_97.py

```
@@ -51,8 +51,20 @@ def test_vip_router_import_error_handling(self):
+        # Test that VIP router handles attribute errors gracefully
+        try:
+            from app import vip_router
+
+            # Try to access a non-existent attribute
+            _ = vip_router.non_existent_attribute
+            # If we get here, the test should fail
+            assert False, "Expected AttributeError was not raised"
+        except AttributeError:
+            # This is expected behavior
+            pass
+        except Exception as e:
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)